hw/ip/otbn/dv/model/iss_wrapper.cc

```diff
@@ -535,6 +535,13 @@ void ISSWrapper::send_err_escalation(uint32_t err_val) {
   run_command(oss.str(), nullptr);
 }
 
+void ISSWrapper::send_rma_req() {
+  std::ostringstream oss;
+  oss << "send_rma_req "
+      << "\n";
```

Comment:
Can this be simplified to
```cpp
oss << "send_rma_req\n";
```
?

Reply:
That was a clang formatter change, I don't exactly know why it complained about this actually

Reply:
Could it be that you wrote
```cpp
oss << "send_rma_req " << "\n";
```
which caused Clang to auto-format on two lines? That could still be simplified into a single string